CMakeLists.txt

```diff
@@ -364,12 +360,43 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/builds/cmake/Modules"
 include(ConfigureWindows)
 include(PlayerFindPackage)
 
+# Platform setup
+set(PLAYER_TARGET_PLATFORM "SDL2" CACHE STRING "Platform to compile for. Options: SDL2 libretro")
+set_property(CACHE PLAYER_TARGET_PLATFORM PROPERTY STRINGS Default libretro)
```

Comment:
↑ What is this supposed to do?

Reply:
That's for the selection boxes in ccmake / cmake-gui but I got the names wrong, Default must be SDL2.